doc/release-notes/rl-2505.section.md

```diff
@@ -36,6 +36,8 @@
 
 - NetBox version 4.0.X available as `netbox_4_0` was removed. Please upgrade to `4.2`.
 
+- `golangci-lint` has reached `v2`. Please read the changes and view the migration guide [here](https://golangci-lint.run/product/changelog/#200).
```

Comment:
That is the changelog link. Maybe we should link that and the migration guide https://golangci-lint.run/product/migration-guide/#command-migrate .

Reply:
The changelog links to the migration guide. I'd rather people view the changelog generally then the migration guide.

Also my concern is the migration guide link may change when a v3 comes out but this changelog item shouldn't ever change.

LMK what you think

Reply:
Fine for me :)